wikimedia/wikipedia-ios
# UI touchups YIR

**Phabricator:** 
https://phabricator.wikimedia.org/T376071
https://phabricator.wikimedia.org/T375566

### Notes
* Bundled both tickets
* Moved out some logic from slideshow
* Created first slide file (very similar to onboarding)

### Test Steps
1. Put in It/Fr, location in It/Fr
2. Navigate to YiR
3. Ensure navigation dots work, have correct colors, are horizontally aligned, animation is correct, and scrollview is correct

### Screenshots/Videos

https://github.com/user-attachments/assets/eb9914a8-560c-4812-ba18-0ca176a37119
